PetProject/Views/LoginScreen/LoginViewController.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  LoginViewController.swift
+//  PetProject
+//
+//  Created by Тимофей Лукашевич on 24.03.22.
+//
+
+import UIKit
+import Combine
+
+protocol LoginViewControllerDelegate: AnyObject, Coordinator {
```

Comment:
Why does this protocol inherit the coordinator protocol?

Reply:
Done